packages/fxa-auth-server/lib/error.js

```diff
@@ -1072,18 +1073,27 @@ AppError.unknownSubscriptionPlan = (planId) => {
   });
 };
 
-AppError.rejectedSubscriptionPaymentToken = (token) => {
+AppError.rejectedSubscriptionPaymentToken = (token, message) => {
   return new AppError({
     code: 400,
     error: 'Bad Request',
     errno: ERRNO.REJECTED_SUBSCRIPTION_PAYMENT_TOKEN,
-    message: 'Rejected subscription payment token'
+    message
   }, {
     token
   });
 };
 
-AppError.subscriptionAlreadyCancelled = (token) => {
+AppError.rejectedCustomerUpdate = (message) => {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.REJECTED_CUSTOMER_UPDATE,
+    message
```

Comment:
This is mostly the point of adding the new error - this message gives more detail from subhub & stripe on exactly why the payment update failed.